ai2_internal/timo/config.yaml

```diff
@@ -0,0 +1,34 @@
+config_version: 0.0.1
+model_variants:
+  vila:
+    # Class path to pydantic Instance implementation in <model_package_name>==<model_package_version>
+    instance: ai2_internal.timo.vila.interface.Instance
+
+    # Class path to pydantic Prediction implementation in <model_package_name>==<model_package_version>
+    prediction: ai2_internal.timo.vila.interface.Prediction
+
+    # Class path to Predictor implementation in <model_package_name>==<model_package_version>
+    predictor: ai2_internal.timo.vila.interface.Predictor
+
+    # Class path to pydantic PredictorConfig implementation in <model_package_name>==<model_package_version>
+    predictor_config: ai2_internal.timo.vila.interface.PredictorConfig
+
+    # Any additional sets of dependencies required by the model.
+    # These are the 'extras_require' keys in your setup.py.
+    extras_require: ["dev", "vila_predictors"]
+
+    # Full S3 path to tar.gz'ed artifacts archive, nullable
+    artifacts_s3_path: s3://ai2-timo-registry/model-artifacts/vila/v0.tar.gz
+
+    # Version of python required for model runtime, e.g. 3.7, 3.8, 3.9
+    python_version: 3.8
+
+    cuda: False
```

Comment:
can explain why false? what does this do?

Reply:
If set to True it'll build a docker image that can leverage a GPU through nvidia-docker. I believe @rodneykinney found that GPU operations accounted for a minority of inference time when he was operationalizing this previously with s2agemaker (which is the same underlying set of templates). Turning this off builds a much smaller and simpler docker image.

Reply:
What I remember is that most of the time was spent in `convert_document_page_to_pdf_dict`. I suspect that method can be made more efficient, possibly at the cost of making the input data shape less convenient. But my conclusion was that with the current code, using a GPU is not worth the cost.